icu4c/source/i18n/unitsdata.cpp

```diff
@@ -358,6 +352,29 @@ int32_t getPreferenceMetadataIndex(const MaybeStackVector<UnitPreferenceMetadata
 
 } // namespace
 
+CharString U_I18N_API getUnitCategory(const char *baseUnitIdentifier, UErrorCode &status) {
```

Comment:
@hugovdm , why when I merged sffc/units-staging I get this extra definition for getUnitCategory() ???

which one do I need to remove? because there is duplication right now

Reply:
I would keep the one from units-staging.

Reply:
Looking at the git commit history:

I last merged units-staging into units-router at bbb7f843e9 on 12 May, and I see units-staging had further changes since then ("unit_header" 6d78a95d6d, "usagefix" 6b14d7f1a0, and "getUnitCategory" d406b915c4).

24494d985e is a common ancestor of the latter two, but not of the unit_header branch. I'm thus wondering if this merge confusion is due to unit_header's age: the two ancestors of a2318015e7 is much further back than that common ancestor commit (24494d985e).